docs/run.js

```diff
@@ -31,30 +28,37 @@ const scalingFactor = 100;
 let resolvedScoreAsset;
 
 /**
- * Sets {@link scorePromise} to the {@link ee.FeatureCollection} with
- * id {@link getScoreAssetPath}, or, if that does not exist, with id
- * {@link getBackupScoreAssetPath}.  Sets {@link resolvedScoreAsset} to the id
- * of the found collection.
+ * Seeds {@link Promise} cache with `eeAsset`, and returns {@link Promise} whose
+ * value is `eeAsset`. Caller can then wait on returned {@link Promise} to see
+ * if `eeAsset` is valid asset.
+ * @param {string} eeAsset
+ * @return {Promise<string>} {@link Promise} with `eeAsset` once {@link
+ *     ee.FeatureCollection} with path `eeAsset` has been downloaded
+ */
+function createPromiseWithPathIfSuccessful(eeAsset) {
+  return getEePromiseForFeatureCollection(eeAsset).then(() => eeAsset);
+}
+
+/**
+ * Sets {@link resolvedScoreAsset} to {@link getScoreAssetPath}, or, if that
```

Comment:
super nit: I \*\*think\*\* @link is for methods/classes and @code is for variables? At least that's how I've always used them, would be interested to hear if that's not how you use them
https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html